crates/miden-lib/asm/account_components/rpo_falcon_512_procedure_acl.masm

```diff
@@ -62,7 +68,27 @@ export.auth__tx_rpo_falcon512_procedure_acl
         dup.1 neq.0
         # => [should_continue, was_any_called', i-1, pad(16)]
     end
+    # => [was_any_called, i-1, pad(16)]
+
+    exec.tx::get_output_notes_commitment
+    # => [OUTPUT_NOTES_COMMITMENT, was_any_called, i-1, pad(16)]
+
+    padw eqw push.0 eq movdn.8 dropw dropw
+    # => [were_output_notes_created, was_any_called, i-1, pad(16)]
+
+    loc_load.0
+    # => [allow_unauthorized_output_notes, were_output_notes_created, was_any_called, i-1, pad(16)]
 
+    not
+    # => [!allow_unauthorized_output_notes, were_output_notes_created, was_any_called, i-1, pad(16)]
+
+    # were_output_notes_created && !allow_unauthorized_output_notes
+    and
+    # => [require_auth, was_any_called, i-1, pad(16)]
+
+    if.true
+        exec.::miden::contracts::auth::basic::auth__tx_rpo_falcon512
+    end
```

Comment:
This would result in verifying the signature twice (it happens again below in case trigger procedures were called)